pages/en/lb4/Server.md

```diff
@@ -15,17 +15,14 @@ The [Server](https://apidocs.strongloop.com/@loopback%2fcore/#Server) interface
 
 ## Usage
 
-LoopBack 4 currently offers the [`@loopback/rest`](https://github.com/strongloop/loopback-next/tree/master/packages/rest) package out of the box which provides an HTTP based server implementation handling requests over REST called `RestServer`. In order to use it in your application, all you need to do is register the `RestComponent` to your application class, and it will provide you with an instance of RestServer listening on port 3000. The following shows how to make use of it:
+LoopBack 4 currently offers the [`@loopback/rest`](https://github.com/strongloop/loopback-next/tree/master/packages/rest) package out of the box which provides an HTTP based server implementation handling requests over REST called `RestServer`. In order to use it in your application, all you need to do is have your application class extend from `RestApplication`, and it will provide you with an instance of RestServer listening on port 3000. The following shows how to make use of it:
```

Comment:
I would change `... your application class extend from RestApplication`. to `... your application class extend RestApplication`. (deleted the `from`).

Reply:
And can you split this line into multiple lines (80 characters) ... makes it easier to leave comments on smaller lines. When strong-docs renders it, it automatically merges the lines. 

Ex:
```
hello
world
```

will render as `hello world`.